fullcalendar-rightclick.js

```diff
@@ -18,11 +18,13 @@
 		};
 		View.prototype.registerRightclickListener = function() {
 			var that = this;
+			// FullCalendar > 3.0.1:
+            var trigger = typeof that.publiclyTrigger === 'function' ? 'publiclyTrigger' : 'trigger';
```

Comment:
This line uses spaces for indentation. Could you replace by tabs? Then I'll merge the PR. Many thanks!